Taxi/Taxi/Presenter/CellView.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  CellView.swift
+//  Taxi
+//
+//  Created by Yosep on 2022/06/09.
+//
+import SwiftUI
+
+struct CellView: View {
+    var body: some View {
+           // Color(red: 248 / 255, green: 248 / 255, blue: 248 / 255, opacity: 1.0)
+            VStack(alignment: .leading, spacing: 13) {
+                HStack {
+                    Spacer().frame(width: 6)
+                    MeetingTimeCell()
+                    Spacer().frame(width: 130)
+                    CurrentUserCell()
+                        .padding(EdgeInsets(top: 0, leading: 15, bottom: 10, trailing: 1))
+                }
+                DestinationCell()
+            }
```

Comment:
Spacer는 차지할 수 있는 최대 크기를 차지하게 하는 용도로 쓰입니다!
따라서 Spacer에는 웬만하면 frame을 적용하지 않습니다.

여기에서 HStack 안에 쓰인 Spacer는 간격을 위해 쓰인 것 같은데, Spacer 대신 padding을 사용하는 것을 권장드립니다.

```suggestion
            VStack(alignment: .leading, spacing: 13) {
                HStack {
                    MeetingTimeCell()
                    Spacer() // 시간과 인원수 사이를 벌어지게 함
                    CurrentUserCell()
                        .padding(EdgeInsets(top: 0, leading: 15, bottom: 10, trailing: 1))
                }
                DestinationCell()
            }
            .padding() // 셀 컨텐츠를 RoundedButton와 마진을 두고 있게 함 -> 위에서 쓰인 Spacer의 간격이 알아서 조정됨
```

Reply:
참고해서 수정하였습니다~!